platform/android/java/plugins/godotpaymentv3/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.library'
+
+android {
+	compileSdkVersion versions.compileSdk
+	buildToolsVersion versions.buildTools
+
+    defaultConfig {
+		minSdkVersion versions.minSdk
+		targetSdkVersion versions.targetSdk
+    }
+
+	libraryVariants.all { variant ->
+		variant.outputs.all { output ->
+			output.outputFileName = "GodotPaymentV3.${variant.name}.aar"
```

Comment:
BTW I'm not sure why this is called `V3` in the first place, but if we're breaking compat we might as well drop the suffix. This versioning probably predates the open sourcing of Godot, unless it actually refers to some Google V3 API?